docs-ref-mapping/reference-latest.yml

```diff
@@ -468,6 +468,7 @@
       - tools.com.microsoft.azure.sdk.iot*
   - name: Key Vault
     uid: azure.java.sdk.landingpage.services.keyvault
+    href: ~/docs-ref-services/keyvault.md
```

Comment:
This is ok, but I already expect it to be here. This is present in reference-unified right? 

The fact that it's not aready here seems like a bug in the ToC sync to me.

Reply:
Yeah I thought that was probably a bug as well. Should I add this reference on `reference-unified` too?